src/db/migrations/v1.9.0-normalise-foreign-keys.js

```diff
@@ -0,0 +1,142 @@
+"use strict";
+
+const dropEstablishmentsForeignKey = (queryInterface, t) =>
+  queryInterface.removeConstraint(
+    "establishments",
+    "establishments_registrationId_fkey",
+    { transaction: t }
+  );
+
+const recreateEstablishmentsForeignKey = (queryInterface, t, setting) =>
+  queryInterface.addConstraint("establishments", ["registrationId"], {
+    type: "foreign key",
+    name: "establishments_registrationId_fkey",
+    references: {
+      table: "registrations",
+      field: "id"
+    },
+    onUpdate: "cascade",
+    onDelete: setting,
+    transaction: t
+  });
+
+const dropMetadataForeignKey = (queryInterface, t) =>
+  queryInterface.removeConstraint("metadata", "metadata_registrationId_fkey", {
+    transaction: t
+  });
+
+const recreateMetadataForeignKey = (queryInterface, t, setting) =>
+  queryInterface.addConstraint("metadata", ["registrationId"], {
```

Comment:
is there a reason why some are square bracketed and others aren't. Not a big deal I'm just curious if related to Postgres casing rules or something.

Reply:
As per Sequelize documentation, the second argument of addConstraint method is: "Array of column names to apply the constraint over".